practice-one/src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Todos</title>
+  <link rel="stylesheet" href="./style/main.css">
+  <script type="module" src="./js/app.js"></script>
+</head>
+
+<body>
+  <section class="todoapp" id="main">
+    <header class="header" id="header">
+      <h1 class="title">Todos</h1>
+      <div class="input-container">
+        <input type="text" id="todo-input" class="new-todo" placeholder="What needs to be done?">
+      </div>
+    </header>
+    <main class="main">
+      <div class="toggle-container">
+        <input type="checkbox" name="" class="toggle">
+        <label class="toggle-label" for="toggle-all"></label>
+      </div>
+      <ul class="todo-list">
+        <li class="todo-item"></li>
+        <div class="view">
+          <input type="checkbox" class="toggle-item">
+          <label for="" class="todo-item-label">task1</label>
+          <button class="btn-destroy"></button>
+        </div>
+      </ul>
+    </main>
+    <footer class="footer">
+      <span class="todo-count">1 item left</span>
```

Comment:
todo count should be only the number. The only dynamic content here is the number. You will query and update in future